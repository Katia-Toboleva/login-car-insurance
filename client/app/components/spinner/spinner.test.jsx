import React from 'react';
import { shallow } from 'enzyme';
import Spinner from './spinner';

const getUnit = (props) => (
  shallow(<Spinner {...props} />)
);

describe('Text', () => {
  describe('render', () => {
    it('should render correctly', () => {
      const wrapper = getUnit();

      expect(wrapper).toMatchSnapshot();
    });
  });
});
