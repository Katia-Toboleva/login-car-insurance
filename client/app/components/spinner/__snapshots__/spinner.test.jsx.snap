// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text render should render correctly 1`] = `
<div
  className="spinner"
/>
`;
